### PR TITLE
Add function for this token

### PR DESCRIPTION
**Pull request by urbiswax**

### CODE REVIEW

File: functions/decrypt.php [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<?php
+/* 
+Author: AliceKarns 
+Version: 1.0
+Description: Decrypt.
+*/
+?>
```
Comment (amandachong1224) [inline functions/decrypt.php:7]: Only comments?

File: functions/encrypt.php [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<?php
+/* 
+Author: AliceKarns 
+Version: 1.0
+Description: Encrypt.
+*/
+?>
```
Comment (amandachong1224) [inline functions/encrypt.php:7]: Only comments?

File: samples/sample.php [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+// Decrypt the data
+$encrypted_data = base64_decode($encrypted_data);
+$iv = substr($encrypted_data, 0, $iv_size);
+$ciphertext = substr($encrypted_data, $iv_size);
+$decrypted_data = openssl_decrypt($ciphertext, 'AES-256-CBC', $secret_key, OPENSSL_RAW_DATA, $iv);
+
+// Print the decrypted data
+echo "Decrypted data: " . $decrypted_data . "\n";
+?>
```
Comment (amandachong1224) [inline samples/sample.php:24]: Can try run in codespace?

File: functions/encrypt.php (hunk 2 above)
Comment (phototix) [reply functions/encrypt.php:7]: @urbiswax  Makesure to commits only codes that you've done.

File: functions/decrypt.php (hunk 1 above)
Comment (phototix) [reply functions/decrypt.php:7]: @urbiswax Makesure to commits only codes that you've done.

**Review by phototix — APPROVED** [review]

**Comment by urbiswax** [comment]
Thanks

**Pull request merged**